src/modules/remote-config/plugins/useFirebaseConfig.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@tanstack/react-query';
+import type { RemoteConfig } from '../types';
+import { fetchRemoteConfig } from './firebase';
+
+export function useFirebaseConfig<T extends keyof RemoteConfig>(
```

Comment:
it looks like I've already seen this function extracted in one of @vyorkin 's prs. Someone will need to merge carefully :))

Reply:
Fortunately it seems that we've created different files so they do not conflict :)